notmuch-addrlookup.c

```diff
@@ -96,24 +96,31 @@ load_notmuch_settings (void)
   gchar *config_path = NULL;
   gboolean success = FALSE;
 
-  if (g_getenv ("NOTMUCH_CONFIG"))
+  if(notmuch_config_path_lookupvar)
+    config_path = notmuch_config_path_lookupvar;
+  else if (g_getenv ("NOTMUCH_CONFIG"))
     config_path = g_strdup (g_getenv ("NOTMUCH_CONFIG"));
-  else if(!notmuch_config_path_lookupvar)
-    config_path = g_strdup_printf ("%s/.notmuch-config", g_get_home_dir ());
+  else if (g_getenv ("NOTMUCH_PROFILE")){
+    if (g_getenv("XDG_CONFIG_HOME")){
+      //TODO check if file exists
+    }
+  }
   else
-    config_path = notmuch_config_path_lookupvar;
+    config_path = g_strdup_printf ("%s/.notmuch-config", g_get_home_dir ());
```

Comment:
I know the path was already being calculated in this way, but it would be better to use [g_build_filename()](https://docs.gtk.org/glib/func.build_filename.html) instead:

```suggestion
    config_path = g_build_filename (g_get_home_dir (), ".notmuch-config", NULL);
```